kikit/fab/neodenyy1.py

```diff
@@ -0,0 +1,164 @@
+import click
```

Comment:
Since the format isn't common (e.g. like gerbers are) nor straightforward, I would appreciate adding a docstring with a simplified format description and links to relevant sources. Otherwise, this becomes "write-only" code that nobody will be able to maintain.

Reply:
Gerbers generation is removed, docstrings updated. I also provide a Markdown doc in docs folder. I will upload the Demo file here.

[0603Demo.csv](https://github.com/yaqwsx/KiKit/files/11988654/0603Demo.csv)